setup.py

```diff
@@ -19,7 +19,7 @@
     },
     install_requires=[
         'argparse==1.3.0',
-        'PyGithub>=1.25.2',
+        'PyGithub==1.26.1',
```

Comment:
this should probably be `>=`. `github-pr` is a library. it shouldn't require specific versions because then all users of the library would need to use those specific version as well